hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/shell/OzoneAddress.java

```diff
@@ -200,12 +200,22 @@ public OzoneClient createClientForS3Commands(
     } else {
       // If om service id is not specified, consider it as a non-HA cluster.
       // But before that check if serviceId is defined. If it is defined
-      // throw an error om service ID needs to be specified.
+      // and has length = 1, then take that ID as default.
+      // otherwise throw an error om service ID needs to be specified.
       if (OmUtils.isServiceIdsDefined(conf)) {
-        throw new OzoneClientException("Service ID must not"
-            + " be omitted when " + OZONE_OM_SERVICE_IDS_KEY + " is defined. " +
-            "Configured " + OZONE_OM_SERVICE_IDS_KEY + " are " +
-            conf.getTrimmedStringCollection(OZONE_OM_SERVICE_IDS_KEY));
+        Collection<String> serviceIds=conf.
+            getTrimmedStringCollection(OZONE_OM_SERVICE_IDS_KEY);
+        if (serviceIds.size() == 1) {
```

Comment:
Maybe make it >=1, else if there arise a case when someone has configured more than one serviceID, this will go to the else statement. 
Open for discussion here

Reply:
I think that's exactly the purpose: if there are multiple OM services, we should not pick one randomly here.

Reply:
Or as we discussed, throw some new exception for >1 case.

Reply:
> I think that's exactly the purpose: if there are multiple OM services, we should not pick one randomly here.

Yeah make sense